tests/integration_tests/albumentations_test.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+import numpy as np
+
+from super_gradients.training.datasets import Cifar10, Cifar100, ImageNetDataset
+from albumentations import Compose, HorizontalFlip, InvertImg
+
+
+class AlbumentationsIntegrationTest(unittest.TestCase):
+    def _apply_aug(self, img_no_aug):
+        pipe = Compose(transforms=[HorizontalFlip(p=1.0), InvertImg(p=1.0)])
+        img_no_aug_transformed = pipe(image=np.array(img_no_aug))["image"]
+        return img_no_aug_transformed
+
+    def test_cifar10_albumentations_integration(self):
```

Comment:
Let's add some tests for segmentation & detection (even if they fails) to sake of completeness. 
I may help with making them pass next week :)

Reply:
I don't see what's the point if they are not supported at the moment. I think its best practice that everyone test their own code and not something planned for someone else to do in the future.

Reply:
Well, I respectfuly disagree. We know as a matter of fact that we will it for segmentation and detection tasks. So, initial implementation of the plugin should support it straight away on the level of API design.